lib/docs/filters/pig/clean_html.rb

```diff
@@ -0,0 +1,26 @@
+module Docs
+  class Pig
+    class CleanHtmlFilter < Filter
+      def call
+        @doc = at_css('#content')
```

Comment:
When you don't need to entire `<body>` to funnel through the other filters (e.g. to better identify the `type` in `EntriesFilter`), you can do this via `options[:container] = '#content'` in the scraper class.